src/sympc/grads/grad_functions.py

```diff
@@ -135,6 +135,12 @@ def forward(ctx: Dict[str, Any], x: MPCTensor, y: Any) -> MPCTensor:
         Returns:
             x + y (MPCTensor): The result of the addition
         """
+        if not hasattr(x, "shape"):
```

Comment:
Insert meme with "we do not do that here"

Reply:
Can't you leave it only for the tensor case at the moment? this should be treated better

Reply:
Nope, it can't be only for tensor case. The reciprocal function uses addition with an integer. So this throws an error saying _"Integer has no attribute shape"_

Reply:
I can just cast the integers in reciprocal as torch tensors and add it instead.

Reply:
Done